sdcm/nemesis.py

```diff
@@ -1548,6 +1548,66 @@ def disrupt_network_block(self):
         finally:
             self.target_node.traffic_control(None)
 
+    def disrupt_cluster_down_scaling(self):
+        """
+        https://docs.scylladb.com/operating-scylla/procedures/cluster-management/remove_node/
+
+        1.terminate
+        2.Full repair
+        3.Nodetool removenode
+        4.Sleep 10 minutes
+        5.Add new node
+        """
+
+        # pick random node
+        self._set_current_disruption('ScaleDown')
+        node = self.target_node
+        up_nodes = self.cluster.get_nodes_up_and_normal(verification_node=node)
+
+        # pick a random node to remove (should be different than node)
```

Comment:
why, this is a problematic choice, since you might pick wrongly
* a seed node, and it might break
* a node that has other nemesis running on them

Reply:
In order to prevent problems you mentioned above, Can I check both `running_nemesis` and `is_seed()`?

Or it's preferable to removing the `target_node` instead?

Reply:
I think it would be better to remove target_node, and do the command on a random other node.

target_node is the affect node by a nemesis, in most cases it's also the node we are issuing commands too. this case is a bit different, since you need to run one of the command one a different node.

But the same general logic apply, and target_node should be remove (and the logic of selecting a node to run nodetool on, is easier)

Reply:
Ok, I eventually selected that randomly, will change that...
Can a target_node also be a seed node? 
After removing a target node, should I randomly select a new one, or it's done automatically in test setup?